### PR TITLE
[release-1.0.0-RC3] Update rights doc

### PR DESCRIPTION
**Pull request by ballinette**
* Fix format for createProfile requests after role refactoring
*  	add the documentation for getUserRights, getProfileRights and getMyRights actions

### CODE REVIEW

File: source/includes/_securityController.md [hunk 1]
```diff
@@ -1610,7 +1774,11 @@ Retrieves all the users matching the given filter.
 <section class="rest"></section>
 ```litcoffee
 {
-    "roles": ["some", "roles"]
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 2]
```diff
@@ -1624,7 +1792,11 @@ Retrieves all the users matching the given filter.
   action: "updateProfile",
   _id: "<profile id>",
   body: {
-    "roles": ["some", "roles"]
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 3]
```diff
@@ -256,10 +271,15 @@ The `user` is created if it does not exists yet or replaced with the given objec
...
-  _id: "<the role id>",
-  // the role definition
+  _id: "<the profile id>",
+  // the profile definition
   body: {
-    roles: ['<roleID>', '<roleID>', ...]
+    // The new array of role IDs and restrictions (cannot be empty)
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 4]
```diff
@@ -239,7 +249,12 @@ The `user` is created if it does not exists yet or replaced with the given objec
 {
   _id: "<profile ID>",
   body: {
-    roles: ['<roleID>', '<roleID>', ...] // The new array of role IDs (cannot be empty)
+    // The new array of role IDs and restrictions (cannot be empty)
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 5]
```diff
@@ -28,10 +33,15 @@
...
-  _id: "<the role id>",
-  // the role definition
+  _id: "<the profile id>",
+  // the profile definition
   body: {
-    roles: ['<roleID>', '<roleID>', ...]
+    // The new array of role IDs and restrictions (cannot be empty)
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 6]
```diff
@@ -11,7 +11,12 @@
 ```litcoffee
 {
   body: {
-    roles: ['<roleID>', '<roleID>', ...] // The new array of role IDs (cannot be empty)
+    // The new array of role IDs and restrictions (cannot be empty)
+    roles: [
+      {_id: '<roleID>'},
+      {_id: '<roleID>', retrictedTo: [{index: '<index>'}, {index: '<index>', collections: ['<coll1>', '<coll2>'] } ] },
```
Comment (scottinet) [inline source/includes/_securityController.md]: re**s**trictedTo

File: source/includes/_securityController.md [hunk 7]
```diff
@@ -791,6 +811,82 @@ Given a `user id`, deletes the corresponding `user` from Kuzzle's database layer
 ```
 Given a `profile id`, retrieve the corresponding profile from the database.
```
Comment (benoitvidis) [inline source/includes/_securityController.md]: retrieve**s**

**Comment by dbengsch** [comment]
+1

**Pull request merged**